src/Graphics/src/Graphics/MathF.cs

```diff
@@ -0,0 +1,35 @@
+#if NETSTANDARD2_0
+namespace System;
+
+internal static class MathF
```

Comment:
For this, should we just use the source code for `MathF`?

* https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/MathF.cs,89a6609942c74f6e
* https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/MathF.cs

You could also only include the functions actually used?

Reply:
I tried that but those files reference IntrinsicAttribute, which is an internal source generator in that codebase. But I do that and then just remove that attribute.

Reply:
In any event, I removed the unused functions

Reply:
I would think removing `IntrinsicAttribute` would be fine, only real concern is to put the same comments/heading at the top and link to the original file.

Reply:
The only thing that I could take from the source code was the AggressiveInlining hints and the approximate const float of PI

Everything else relied on the Intrinsics namespace that is all internal.